services/backend-api/scripts/scalping-soak-acceptance_test.sh

```
@@ -240,6 +247,8 @@ RUN_HEALTH_PREFLIGHT=false \
   CHECK_GATEWAY_STATUS=false \
   MAX_HOLD_RATIO= \
   EXPECTED_MAX_HOLD_RATIO= \
+  MAX_PERFECT_WIN_TRADES= \
+  EXPECTED_MAX_PERFECT_WIN_TRADES= \
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)